src/components/revurdering/oppsummering/formuevilkåroppsummering/FormuevilkårOppsummering.tsx

```diff
@@ -23,66 +22,68 @@ const FormuevilkårOppsummering = (props: { gjeldendeFormue: FormueVilkår }) =>
 );
 
 export const Formuevurdering = ({ vurdering }: { vurdering: VurderingsperiodeFormue }) => {
-    const { intl } = useI18n({ messages });
+    const { formatMessage } = useI18n({ messages });
     const søker = vurdering.grunnlag.søkersFormue;
     const eps = vurdering.grunnlag.epsFormue;
 
     return (
-        <div className={styles.oppsummeringsContainer}>
+        <div>
             <OppsummeringPar
                 className={styles.gjeldendePeriode}
-                label={intl.formatMessage({ id: 'gjeldendeformue.gjeldendePeriode' })}
+                label={formatMessage('gjeldendeformue.gjeldendePeriode')}
                 verdi={DateUtils.formatPeriode(vurdering.periode)}
             />
-            <div className={styles.titler}>
-                <BodyShort>{intl.formatMessage({ id: 'gjeldendeformue.søker' })}</BodyShort>
-                {eps && <BodyShort>{intl.formatMessage({ id: 'gjeldendeformue.eps' })}</BodyShort>}
-            </div>
             <OppsummeringPar
                 className={styles.oppsummeringstriple}
-                label={intl.formatMessage({ id: 'gjeldendeformue.verdiBolig' })}
+                label={''}
```

Comment:
Riktig at label skal være tom?

Reply:
Yep, slipper å ha spesiell håndtering for at den bare vises i 2 av 3 kolonnene